src/app/systelab-translate/date-util/date-util.ts

```diff
@@ -123,26 +95,6 @@ export class DateUtil {
 			case 'ja-JA':
 				stringDateFormat = 'YY/MM/DD';
 				break;
-			default:
-				stringDateFormat = 'DD/MM/YY';
-				break;
-		}
-		if (isFullYear) {
-			stringDateFormat = stringDateFormat.replace('YY', 'YYYY');
-			if (this.locale === 'en-US') {
-				stringDateFormat = stringDateFormat.replace('M', 'MM');
-				stringDateFormat = stringDateFormat.replace('D', 'DD');
-			}
-		}
-		return stringDateFormat;
-	}
-
-	public getDateFormatForDatePicker(isFullYear = false): string {
-		let stringDateFormat = '';
-		switch (this.locale) {
-			case 'en-US':
-				stringDateFormat = 'm/d/y';
-				break;
 			case 'en-GB':
```

Comment:
As done in getDateFormatForDatePicker,
It would be better to list before the "defalt clause", the "case of each locale" which gonna work with the default format

they was previously removed
			case 'en-GB':	
			case 'it-IT':	
			case 'es-AR':	
			case 'es-ES':	
			case 'es-BO':	
			case 'es-CL':	
			case 'es-CO':	
			case 'es-CR':	
			case 'es-DO':	
			case 'es-EC':	
			case 'es-GT':	
			case 'es-HN':	
			case 'es-MX':	
			case 'es-NI':	
			case 'es-PA':	
			case 'es-PE':	
			case 'es-PR':	
			case 'es-PY':	
			case 'es-SV':	
			case 'es-UR':	
			case 'es-VE':	
			case 'fr-FR':	
			case 'gl-GL':	
			case 'ca-CA':